subprojects/gradle/build-verdict/src/main/kotlin/com/avito/android/build_verdict/internal/BuildVerdictPluginServices.kt

```diff
@@ -21,9 +21,10 @@ import java.util.concurrent.ConcurrentHashMap
 
 internal class BuildVerdictPluginServices(
     private val extension: BuildVerdictPluginExtension,
-    private val logger: CILogger
+    loggerFactory: LoggerFactory
 ) {
 
+    private val logger = loggerFactory.create("BuildVerdictPluginServices")
```

Comment:
ClassName should be `BuildVerdict`

Reply:
i don't see why. it's not a className then, just a tag
BuildVerdictPluginServices is a className when this logger created

Reply:
renamed argument to `tag`

Reply:
> > no more files and stdout logs in CI
> 
> Will I see logs in stdout in a local build?

yes, it looks like https://github.com/avito-tech/avito-android/pull/695#issuecomment-751751157